Code/GraphMol/MolStandardize/Charge.cpp

```diff
@@ -296,9 +334,25 @@ ROMol *Uncharger::uncharge(const ROMol &mol) {
   }
   unsigned int n_matched = SubstructMatch(*omol, *(this->neg), n_matches);
   unsigned int a_matched = SubstructMatch(*omol, *(this->neg_acid), a_matches);
+  std::vector<int> n_matches_mol;
+  for (const auto &n_match : n_matches) {
+    std::transform(n_match.begin(), n_match.end(),
+                   std::back_inserter(n_matches_mol),
+                   [](const std::pair<int, int> &p) { return p.second; });
+  }
+  std::vector<int> a_matches_mol;
+  for (const auto &a_match : a_matches) {
+    std::transform(a_match.begin(), a_match.end(),
+                   std::back_inserter(a_matches_mol),
+                   [](const std::pair<int, int> &p) { return p.second; });
+  }
+  std::set<int> a_n_matches_union;
```

Comment:
Since you're using `std::set_union` to do the insert, there's no need to pay the additional price of using a set here:
```suggestion
  std::vector<int> a_n_matches_union;
```